tests/clar.c

```diff
@@ -419,37 +423,33 @@ clar_test_run(void)
 {
 	if (_clar.argc > 1)
 		clar_parse_args(_clar.argc, _clar.argv);
-
 	if (!_clar.suites_ran) {
 		size_t i;
 		for (i = 0; i < _clar_suite_count; ++i)
 			clar_run_suite(&_clar_suites[i], NULL);
 	}
-
 	return _clar.total_errors;
 }
 
 void
-clar_test_shutdown(void)
+clar_test_shutdown()
 {
 	clar_print_shutdown(
 		_clar.tests_ran,
-		(int)_clar_suite_count,
-		_clar.total_errors
+		_clar.suites_ran,
+		_clar.total_errors,
+		_clar.total_skipped
```

Comment:
Might be worth it to just pass in the complete `clar` struct?

Reply:
tried to not change how it worked before (additional code churn) but OK, I'll amend it!

Reply:
actually we can't do that, the print.h file where this function is defined does not know about the ``_clar`` struct and its type

Reply:
Okay, fair enough